test/driver.js

```diff
@@ -249,6 +253,10 @@ class Rasterize {
       });
       await annotationLayer.render(parameters);
       await annotationLayer.showPopups();
+      await document.l10n.translateRoots();
```

Comment:
Please add a comment explaining why we do this here.
```suggestion
      // Translation should happen "automatically" in the annotationLayer,
      // however we cannot await that hence we manually trigger translation
      // here to prevent any intermittent test failures.
      await document.l10n.translateRoots();
```